airbyte-webapp/src/components/JobItem/components/Logs.tsx

```diff
@@ -47,8 +46,10 @@ const Logs: React.FC<LogsProps> = ({ logsArray }) => {
           lineClassName="logLine"
           highlightLineClassName="highlightLogLine"
           selectableLines
-          follow
+          follow={true}
           style={{ background: "transparent" }}
+          scrollToLine={undefined}
+          highlight={[]}
```

Comment:
Is this a hack to prevent it from scrolling to a specific line?

Reply:
yep - and it was undocumented and super hard to find.  That's why I left this in so we would remember this for the future when we get to https://github.com/airbytehq/airbyte/issues/12128